index.html

```diff
@@ -1,508 +0,0 @@
-<!DOCTYPE html>
-<html lang="pt-BR">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="css/reset.css" />
-    <script
-      type="module"
-      src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"
-    ></script>
-    <script
-      nomodule=""
-      src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"
-    ></script>
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="css/styles.css" />
-
-    <title>Instagram</title>
-  </head>
-  <body>
-    <div class="website">
-      <!-- inicio topo -->
-      <div class="topo">
-        <div class="nav f y-center x-between">
-          <div class="logo-principal">
-            <a href="" class="f xy-center">
-              <div class="divisor-logo">
-                <ion-icon name="logo-instagram"></ion-icon>
-              </div>
-              <img src="imagens/logo.png" alt="logo-instagram" />
-            </a>
-          </div>
```

Comment:
Uma boa prática na nomeação de classes é usar todos os termos em inglês. Por exemplo, você tem uma <div class="nav">, o que é ótimo, mas logo dentro dela tem uma <div class="logo-principal">.
Não é errado fazer isso, só é mais conveniente manter um estilo padrão pra nomear. Isso ajuda bastante pra quando você for trabalhar com uma equipe de devs, evitando aquela sensação de insegurança sobre "como nomear esse cara", tendo um padrão estabelecido, essa dúvida nunca vai aparecer.
Resumindo, é recomendável usar nomes em inglês nas classes css.